qiskit/pulse/reschedule.py

```diff
@@ -16,8 +16,10 @@
 import warnings
 
 # pylint: disable=unused-import
-
-from .transforms import align_measures, add_implicit_acquires, pad, compress_pulses
+from qiskit.pulse.transforms import (align_measures,
```

Comment:
Can you please stick to these throughout:
```
from qiskit.etc import module0, module1
from qiskit.etcetc import (function, Class0, Class1, Class2,
                           FollowingLine)
```

Other styles I've seen in this PR (some I have not seen in Terra yet):
```
from qiskit.etc import Class0
from qiskit.etc import Class1
from qiskit.etcetc import (
    one,
    two,
    three
)
from qiskit.xx import (first,
                       second,
                       third)
```

We all worked so hard together at the docathon earlier this year to improve the style and documentation of the codebase, and to encourage everyone to be a little more consistent about the little things. At the very least, it gives the code a sense of professionalism.

As a reviewer, it felt like a large portion of the changes to existing files in this PR are updates to import statements and the side effects caused by that, so I hope you don't think this is too nitpicky. I'm ok with importing modules sometimes and functions other times, and I've always preferred absolute imports over relative imports. If you are consistent throughout these 37 files with how you organize your imports, perhaps it'll be more likely that future developers will maintain this style and use it as a template for future work.

Reply:
Another place where a new style is being introduced (and is not maintained everywhere, or recommended by our style guide) is some function signatures, with another new line after the last argument.
ex
```
def function(arg,
             arg2
             ) -> return_type:
```